server/views/components/instagram-thumbnail/index.njk

```diff
@@ -0,0 +1,31 @@
+<a href="{{ model.url }}" class="instagram-thumbnail"
+  data-track-click='{"screenName": "{{ model.screenName }}", "createdAt": "{{ model.createdAt }}", "postId": {{ model.id }}}'>
```

Comment:
I think they all have some merit/can't harm – it's a relatively straightforward step to filter out noise in Mixpanel.